src/Types/IDEntityBody.php

```diff
@@ -55,9 +63,13 @@ public static function normalize($value): ?string
      */
     public function isValid(): bool
     {
-        /** @var BodyCodeValidatorExtension $validator */
-        $validator = static::getContainer()->make(BodyCodeValidatorExtension::class);
+        if (\is_string($this->value) && $this->value !== '') {
+            /** @var BodyCodeValidatorExtension $validator */
+            $validator = static::getContainer()->make(BodyCodeValidatorExtension::class);
+
+            return \is_string($this->value) && $validator->passes('', $this->value);
```

Comment:
Redundant `\is_string($this->value)`. We are check this condition above.

Reply:
Yes, you right!